src/GovUk.Education.ExploreEducationStatistics.Admin/Validators/ValidationErrorMessages.cs

```diff
@@ -113,5 +113,10 @@ public enum ValidationErrorMessages
         // Key statistics
         DataBlockShouldBeUnattached,
         ProvidedKeyStatIdsDifferFromReleaseKeyStatIds,
+
+        // Featured tables
+        DataBlockAlreadyHasFeaturedTable,
+        FeaturedTableIsNotAssociatedWithRelease,
```

Comment:
As mentioned in other places, we should just return a 404 instead so we don't need this custom validation message.